src/main/java/net/zerowatch/OAuthCommand.java

```diff
@@ -0,0 +1,82 @@
+package net.zerowatch;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+/**
+ * Handles oAuth requests.
+ */
+ @RestController
+public final class OAuthCommand {
```

Comment:
What is this command actually used for...? I assume it isn't an actual slash command - is it just used by Slack to do auth, but not exposed?